.travis.yml

```diff
@@ -1,2 +1,2 @@
 script:
-- echo "hello world"
+- docker build . -t travis/image:1.0
```

Comment:
This is too minimal for this exercise I'm afraid, we should add some tests as part of M3 and then come back to this when we're able to run those tests as part of this pipeline